index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0"> <!-- displays site properly based on user's device -->
+
+  <link rel="icon" type="image/png" sizes="32x32" href="./assets/images/favicon-32x32.png">  
+  <link rel="stylesheet" href="styles.css">
+  
+  <title>Frontend Mentor | Social links profile</title>
+</head>
+<body>
+  <div class="wrapper">
+    <img alt="jessica randall headshot" id="headshot" src="./assets/images/avatar-jessica.jpeg">
+    <div class="title">
+      <h1>Jessica Randall</h1>
+      <p>London, United Kingdom</p>
+    </div>
+    
+    <p>"Front-end developer and avid reader."</p>
+  
+    <ul>
+      <li>GitHub</li>
+      <li>Frontend Mentor</li>
+      <li>LinkedIn</li>
+      <li>Twitter</li>
+      <li>Instagram</li>
```

Comment:
Since these are links, you should wrap the text within the `<li>` in an `<a>`. Not only does this make them link out (I know this doesn't provide URLs, so could use dummy URLs), it also makes them keyboard-interactive, which is important for accessibility